README.md

```diff
@@ -1,16 +1,39 @@
-## Threadx Demo
-- Using stmcubeide version: 1.15.1 and an STM32f446RE
-- RTOS code is in Threadx-Demo/Core/Src/app_threadx.c
-- Using 3 threads with normal priority
-- Thread0 
-	- loads a message into a queue
-	- When the message is loaded a thread 0 specific count increments by 1
-	- Thread0 sum is the sum of all the random numbers it sends
-	- Thread0 prints every global piece of data
-- Thread1 and thread2 
-	- Each take the message with the random number they recieved and print it over uart
-	- Thread1 sum is the sum of the numbers it recieves
-	- Shared count is a count of the total messages sent
-	- Shared sum is a sum of all the random numbers sent
-	- When the message is recieved the threads increment a gloabl variable by 1
-	- Thread1 and 2 print their resepective counts and messages recieved over uart
+# ThreadX Demo
+- Using STM32CubeIDE version 1.15.1 and an STM32F446RE
+- RTOS code is in `ThreadX-Demo/Core/Src/app_ThreadX.c`
+- Using 3 threads with normal priority:
+
+The point of this demo is to have an example of how we can create and use threads in ThreadX.
```

Comment:
It is also to test that threadx is able to run correctly on our platform and to have something to compare the behavior to when we add this to EVT-core.